devfiles/src/classes/data/DataFilterer.ts

```diff
@@ -55,6 +55,7 @@ export default class DataFilterer {
     // columnIndex can't be larger than the number of total columns
     // filterClass needs to be correct type, won't be checked here
     public replaceFilter(columnIndex, filterClass: Filter) {
+        this.filterState = uuidv4(); //Filters changed
```

Comment:
Are you sure you don't want to add this to dataUpdated()?

Reply:
dataUpdated has this.dataState = uuidv4